FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m

```diff
@@ -129,4 +203,58 @@ - (void)deliverNotification {
   }
 }
 
+- (void)exportDeliveryMetricsToBigQueryWithMessageInfo:(NSDictionary *)info {
+  GDTCORTransport *transport = [[GDTCORTransport alloc] initWithMappingID:@"1249"
```

Comment:
Is there a reason why we need to create a new instance of `GDTCORTransport` each time the method is called? It's preferable to create a single one in `init` and reuse it.

Reply:
I'm just following ads and crashlytics usage here.